packages/asset-service/src/generateAssetData/ethTokens/uniV2.ts

```diff
@@ -0,0 +1,32 @@
+import { AssetNamespace, toAssetId, toChainId } from '@shapeshiftoss/caip'
+import { AssetDataSource, ChainTypes, NetworkTypes, TokenAsset } from '@shapeshiftoss/types'
+
+export const getUniV2Token = (): TokenAsset[] => {
+  const chain = ChainTypes.Ethereum
+  const network = NetworkTypes.MAINNET
+  const assetNamespace = AssetNamespace.ERC20
+  const assetReference = '0x470e8de2ebaef52014a47cb5e6af86884947f08c' // UNI-V2 contract address
+
+  const result: TokenAsset = {
+    assetId: toAssetId({
+      chain,
+      network,
+      assetNamespace,
+      assetReference
+    }),
+    chainId: toChainId({ chain, network }),
+    dataSource: AssetDataSource.CoinGecko,
```

Comment:
This key should ultimately be removed at some point, but I would just use empty string as the value since we are not using it anywhere and coingecko source is not really accurate

Reply:
Agree, though leave it for now - an empty string won't compile, and I'll be killing `dataSource` entirely in https://github.com/shapeshift/lib/issues/651.